Explorer/Assets/DCL/Passport/Modules/UserBasicInfo_PassportModuleController.cs

```diff
@@ -11,43 +15,59 @@ namespace DCL.Passport.Modules
 {
     public class UserBasicInfo_PassportModuleController : IPassportModuleController
     {
-        private readonly UserBasicInfo_PassportModuleView view;
-        private readonly ISelfProfile selfProfile;
-        private readonly PassportErrorsController passportErrorsController;
+        private const string CLAIM_NAME_URL = "https://decentraland.org/marketplace/names/claim";
+
         private readonly UserNameElementController nameElementController;
         private readonly UserWalletAddressElementController walletAddressElementController;
+        private readonly UserBasicInfo_PassportModuleView view;
+        private readonly ISelfProfile selfProfile;
+        private readonly IMVCManager mvcManager;
+        private readonly INftNamesProvider nftNamesProvider;
+        private readonly bool isNameEditorEnabled;
+
+        private CancellationTokenSource? checkNameEditionCancellationToken;
+        private CancellationTokenSource? showNameEditorCancellationToken;
+        private Profile? currentProfile;
 
-        private Profile currentProfile;
-        private CancellationTokenSource checkEditionAvailabilityCts;
+        public event Action? NameClaimRequested;
 
         public UserBasicInfo_PassportModuleController(
             UserBasicInfo_PassportModuleView view,
             ISelfProfile selfProfile,
-            PassportErrorsController passportErrorsController)
+            IWebBrowser webBrowser,
+            IMVCManager mvcManager,
+            INftNamesProvider nftNamesProvider,
+            bool isNameEditorEnabled)
         {
             this.view = view;
             this.selfProfile = selfProfile;
-            this.passportErrorsController = passportErrorsController;
-
+            this.mvcManager = mvcManager;
+            this.nftNamesProvider = nftNamesProvider;
+            this.isNameEditorEnabled = isNameEditorEnabled;
             nameElementController = new UserNameElementController(view.UserNameElement);
             walletAddressElementController = new UserWalletAddressElementController(view.UserWalletAddressElement);
+
+            view.ClaimNameButton.onClick.AddListener(() =>
+            {
+                webBrowser.OpenUrl(CLAIM_NAME_URL);
+                NameClaimRequested?.Invoke();
```

Comment:
even if it's minor better to extract the lambda content to a method to avoid the lambda allocation